cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -281,6 +255,47 @@ runQueryTip (AnyConsensusModeParams cModeParams) network mOutFile = do
       Aeson.Object $ obj <> HMS.fromList [name .= Aeson.Null]
     toObject _ _ _ = Aeson.Null
 
+queryNodeTip
```

Comment:
I would rename this to something like `queryHistoryAndStart`

Reply:
Done!